tests/source/issue-6202/issue_example.rs

```diff
@@ -0,0 +1,19 @@
+// rustfmt-max_width: 120
+// rustfmt-comment_width: 100
+// rustfmt-match_block_trailing_comma: true
+// rustfmt-wrap_comments: true
+// rustfmt-edition: 2021
+// rustfmt-error_on_line_overflow: true
+// rustfmt-imports_granularity: Module
+// rustfmt-version: Two
```

Comment:
Although the user provided us with all of their configs, I think it's only the `max_width` setting that we need to reproduce the error. Could you trim this down to only include the relevant configs.